docs/glossary.md

```diff
@@ -0,0 +1,38 @@
+# AI Edge Terminology
+
+## Core
+
+- The central OpenShift cluster containing the tools responsible for creating any artifacts required for the successful deployment of an Inference Application to Near Edge environments.
+- There are no resources or network constraints expected in the core cluster as it is expected that it fully supports all workflows required for creating and verifying Inference Application container images.
+
+## Near Edge
+
+- This is a non-core distributed environment to run and serve AI/ML inference workloads in moderate yet constrained compute resources and network.
+- For the purpose of this repository, the near edge environment is represented by separate OpenShift clusters disconnected from the core, the internet or both but may be managed from a core OpenShift cluster.
+
+## Model Server
+
+- A Model Server is responsible for hosting models as a service "to return predictions based on data inputs that you provide through API calls."[^2]
+- For any workflows under opendatahub-io/ai-edge, we will be focusing on using the Model Servers and serving runtimes supported by Open Data Hub
+
+## Inference Application Container
+
+- OCI compliant container image[^3] with the models included during the build process
+- Support for container images where the model and model serving runtimes are stored together
+
+## Model Registry
+
+- A centralized repository for the models and their metadata and managing the model lifecycle and versions. Currently our pipelines only support S3 and Git as the source where models can be stored. 
```

Comment:
My understanding is that Model Registry would store only the model metadata, not the actual models anyway.

So should we clarify that currently our pipelines do not support any Model Registry and locations of models (either in S3 or in git repos) need to be specified directly?

Reply:
So, in this terminology we are trying to define Model Registry as a generic model registry and not the ODH Model Registry. 

> So should we clarify that currently our pipelines do not support any Model Registry and locations of models (either in S3 or in git repos) need to be specified directly?

^This I think is clarified by the last line. `Currently our pipelines only support S3 and Git as the source where models can be stored. ` Since we are trying to keep the terminology generic, we didn't want to mention that we don't support any other registry. Does that make sense?